index.php

```diff
@@ -142,6 +142,12 @@ function get_q_value($mime, $accept)
         error_page('Verification Failed', 'Given Code Was Invalid');
     }
 
+    $response = array('me' => USER_URL);
+
+    if ($scope = (explode(':', $code, 3))[2]) {
```

Comment:
As a general rule, I consider side effects to if statements to be bad practice.  It's not any less efficient to make it multiple lines